themes/default/content/blog/iac-recommended-practices-structuring-pulumi-projects/index.md

```diff
@@ -0,0 +1,105 @@
+
+Here are links to all the blog posts in the series (entries below that are not linked are planned but haven't yet been published---this will get updated as new posts are published):
+
+[IaC Recommended Practices: Code Organization and Stacks](/blog/iac-recommended-practices-code-organization-and-stacks/)
+
+[IaC Recommended Practices: Developer Stacks and Git Branches](/blog/iac-recommended-practices-developer-stacks-git-branches/)
+
+**IaC Recommended Practices: Structuring Pulumi Projects** (this post)
+
+IaC Recommended Practices: Tying Stacks Together with Stack References
+
+IaC Recommended Practices: Local Testing with Pulumi
+
+IaC Recommended Practices: Evolving the Application
+
+IaC Recommended Practices: Adding Pulumi Deployments
+
+IaC Recommended Practices: Refactoring for Reuse
+
+## Catching up with Zephyr
+
+As a result of launching its online presence, Zephyr saw amazing success. The market for arcane artifacts and novel curiosities is booming, and Zephyr is becoming a leader in this market. Funded by this success, Zephyr has grown rapidly. Along with this growth, Zephyr's software engineering and IT teams also grew and expanded. Fortunately, Zephyr's [use of per-developer stacks with short-lived ephemeral Git branches](/blog/iac-recommended-practices-developer-stacks-git-branches/) positioned the development teams well for this growth.
+
+{{% notes %}}
+For a look at the code as it was at the end of the second post in the series, see the [`dev-stacks`](https://github.com/pulumi/zephyr-app/tree/dev-stacks) tag of the [`pulumi/zephyr-app`](https://github.com/pulumi/zephyr-app) repository on GitHub.
+{{% /notes %}}
+
+Naturally this growth has resulted in some separation of duties. The IT team now has responsibility for the infrastructure that supports the online retail store. Within that team, some team members are focused on "core" infrastructure, while other team members are focused only on Kubernetes (a natural decision given Zephyr's use of Kubernetes in their architecture). The application team continues to remain responsible for the online store application itself.
+
+This left Zephyr again asking some questions about the best path forward:
+
+* **Should they use a single project, or multiple projects?** Zephyr had used a single Pulumi project up until now, but growth and organizational change meant they wanted to make sure they were using Pulumi in a way that best reflected their needs. What are the advantages and disadvantages of each approach?
+* **If multiple projects, how many projects?** What's the best way to determine how many projects are needed? What factors play a role in this decision?
+
+## Structuring Pulumi projects
+
+One project, or multiple projects? And if multiple projects, how many projects? These are the core questions the Zephyr needed to resolve. In the end, there are a number of key factors that determine the answers to these questions:
```

Comment:
I might skip this paragraph since it almost directly duplicates the content above.